### DISCUSSION TITLE
Running FATES on Derecho

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#1146] [General]
Has anyone successfully run CLM-FATES on Derecho? The [first ctsm tag supported and tested on Derecho seems to be ctsm5.1.dev158](https://groups.google.com/a/ucar.edu/g/ctsm-dev/c/Em-P_c2akWU), but the latest version of fates with a [listed ctsm compatibilty](https://fates-users-guide.readthedocs.io/en/latest/user/Table-of-FATES-API-and-HLM-STATUS.html) is ctsm5.1.dev153. Just curious if anyone has got this working yet and what versions of ctsm, cime, and fates you've been using.
Adam

**Comment by glemieux** [comment]
Hi @adamhb.  @rgknox and I are running on derecho.  The table you referenced is a little out of date (I'll update it soon).  The latest ctrm-fates api update tag was tested and integrated with derecho ([ctsm5.1.dev160](https://github.com/ESCOMP/CTSM/releases/tag/ctsm5.1.dev160) which corresponds to fates tag [sci.1.70.0_api.32.0.0](https://github.com/NGEET/fates/releases/tag/sci.1.70.0_api.32.0.0)).

**Reply by adamhb** [reply]
OK, thanks @glemieux. And what version of cime are you using with that setup? 

PS Do you know if any prior versions of ctsm are being supported on Derecho? I heard that NCAR was generally not planning to support older versions, but curious if anyone heard that had changed?

**Reply by glemieux** [reply]
I'm using the same cime tag noted in the externals config file, cime6.0.175.  

That's my understanding as well.  The CTSM folks created a [transition document](https://docs.google.com/document/d/1FOluB3C7RTHu8vArWGjACwFjGJfxkLcaGP00l0585_k/edit?usp=sharing) which largely recommends that devs merge their branches up to dev157 and doesn't mention testing older tags.

**Comment by rgknox** [comment]
I recommend running ./manage_externals/checkout_externals and let it checkout cime for you

**Comment by adamhb** [comment]
Ok, thanks @rgknox and @glemieux for these recs and info!
